integration-testing/tests/find-users.test.js

```
@@ -103,3 +104,92 @@ test('Find users by phone, and by phone and email', async () => {
+  })
+
+  // dismiss the card
+  await firstClient
+    .mutate({mutation: mutations.deleteCard, variables: {cardId}})
+    .then(({data}) => expect(data.deleteCard.cardId).toBe(cardId))
+
+  // find different Users with new user
+  await misc.sleep(2000)
+  await ourClient
+    .query({query: queries.findUsers, variables: {emails: [ourEmail, other1Email, other2Email]}})
+    .then(({data: {findUsers}}) => {
+      expect(findUsers.items.map((item) => item.userId).sort()).toEqual(
+        [ourUserId, other1UserId, other2UserId].sort(),
+      )
+    })
+  // check first called user has card
+  await firstClient.query({query: queries.self}).then(({data: {self}}) => {
+    expect(self.userId).toBe(other1UserId)
+    expect(self.cards.items[0].cardId).toBe(`${other1UserId}:NEW_FOLLOWER:${ourUserId}`)
+  })
+  // check second called user has card
+  await secondClient.query({query: queries.self}).then(({data: {self}}) => {
+    expect(self.userId).toBe(other2UserId)
+    expect(self.cards.items[0].cardId).toBe(`${other2UserId}:NEW_FOLLOWER:${ourUserId}`)
+  })
+
+  // find different Users with other new user
+  await misc.sleep(2000)
+  await otherClient
+    .query({query: queries.findUsers, variables: {emails: [otherEmail, other1Email, other2Email]}})
+    .then(({data: {findUsers}}) => {
+      expect(findUsers.items.map((item) => item.userId).sort()).toEqual(
+        [otherUserId, other1UserId, other2UserId].sort(),
+      )
+    })
+  // check first called user has card
+  await firstClient.query({query: queries.self}).then(({data: {self}}) => {
+    expect(self.userId).toBe(other1UserId)
+    expect(self.cards.items[0].cardId).toBe(`${other1UserId}:NEW_FOLLOWER:${otherUserId}`)
+  })
+  // check second called user has card
+  await secondClient.query({query: queries.self}).then(({data: {self}}) => {
+    expect(self.userId).toBe(other2UserId)
+    expect(self.cards.items[0].cardId).toBe(`${other2UserId}:NEW_FOLLOWER:${otherUserId}`)
+  })
+})
```

The integration tests in this file don't pass for me anymore. They all fail. Do they pass for you? Have you been able to run them? They should all pass.

To run the tests in this file, you can run `yarn test tests/find-users.test.js` from the `integration-testing` directory.

Tip: when running the integration tests manually, you can comment out the line near the top `jest.retryTimes(2)` to get them to fail faster.